sound/soc/sof/sof-audio.h

```diff
@@ -491,6 +491,8 @@ struct snd_sof_dai {
 	int number_configs;
 	int current_config;
 	struct list_head list;	/* list in sdev dai list */
+	/* This is intended for selecting widget specific ops for the BE DAI widgets */
+	void *private_ops;
```

Comment:
Can we be more generic and `platform_be_data`, but this will also imply that a given DAI can only use a static parameter?
Since ChainDMA support has been mentioned:
A DAI can be used with ChainDMA or it can not be used with ChainDMA? We can not select 'low latency'/'passthrough' (ChainDMA) PCM and a processed or enhanced (DSP processing) PCM towards the same DAI like we do with normal/DeepBuffer?